lib/omise/json.ex

```diff
@@ -0,0 +1,107 @@
+defmodule Omise.Json do
+  alias Omise.Json.{Decoder, Encoder}
+
+  defdelegate encode!(input), to: Encoder
+  defdelegate decode(input, opts \\ []), to: Decoder
```

Comment:
Should provide the same interfaces for encode/decode for OmiseJson module? bang vs non-bang